packages/cfd/src/Containers/dynamic-leverage/dynamic-leverage-market-card.tsx

```diff
@@ -0,0 +1,89 @@
+                    align='center'
+                    className='dynamic-leverage-modal__market-example'
+                >
+                    {localize(market_example)}
+                </Text>
+            ) : null}
+            <Text data-testid='leverage_title' size='xxs' color='red' align='center'>
+                {localize(leverage)}
+            </Text>
+        </div>
+        <Table className='dynamic-leverage-modal__market-table'>
+            <Table.Header>
+                <Table.Row className='dynamic-leverage-modal__market-table-header-row'>
+                    <Table.Head>
+                        <div className='dynamic-leverage-modal__market-table-header'>
+                            <Text size='xs' align='center' weight='bold'>
+                                {localize('From')}
+                            </Text>
+                            <Text size='xxs' align='center'>
+                                {localize('(lots)')}
+                            </Text>
+                        </div>
+                    </Table.Head>
+                    <Table.Head>
+                        <div className='dynamic-leverage-modal__market-table-header'>
+                            <Text size='xs' align='center' weight='bold'>
+                                {localize('to')}
+                            </Text>
+                            <Text size='xxs' align='center'>
+                                {localize('(lots)')}
+                            </Text>
+                        </div>
+                    </Table.Head>
+                    <Table.Head>
+                        <div className='dynamic-leverage-modal__market-table-header'>
+                            <Text size='xs' align='center' weight='bold'>
+                                {localize('Leverage')}
+                            </Text>
+                            <Text size='xxs' align='center'>
+                                {localize('(1:x)')}
+                            </Text>
+                        </div>
+                    </Table.Head>
+                </Table.Row>
+            </Table.Header>
+            <Table.Body>
+                {dynamicLeverages.map((x, index) => (
+                    <Table.Row key={index} className='dynamic-leverage-modal__market-table-row'>
+                        <Table.Cell>
+                            <Text size='xs' align='center'>
+                                {x.from}
+                            </Text>
+                        </Table.Cell>
+                        <Table.Cell>
+                            <Text size='xs' align='center'>
+                                {x.to}
+                            </Text>
+                        </Table.Cell>
+                        <Table.Cell>
+                            <Text size='xs' align='center'>
+                                {x.leverage}
+                            </Text>
+                        </Table.Cell>
```

Comment:
All table cells have same style. can we use map to show to, from and leverage?

Reply:
Updated.